src/components/PostCard/EventBody/EventBody.test.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { shallow } from 'enzyme'
 import EventBody from './EventBody'
-import moment from 'moment'
+import moment from 'moment-timezone'
```

Comment:
Standardized on the use of `moment-timezone` throughout so we don't have to explicitly require `moment` in dependencies. This is true in `hylo-shared` as well which relies upon `moment-timezone` as a `peerDependency` so ultimately this keeps versions aligned and will help with bundle management when we go another round on that. 

* `moment-timezone` is still the first place to look for reducing our bundle size and there are a couple more modern feature compatible libraries to look into which are smaller and, nicer, if we want to switch over at some point.